product_docs/docs/pgd/5/parallelapply.mdx

```diff
@@ -61,9 +62,13 @@ Up to and including PGD 5.1, Parallel Apply should not be used with Group Commit
 
 ### Monitoring Parallel Apply
 
-To support Parallel Apply's deadlock mitigation, PGD 5.2 adds columns to [`bdr.stat_subscription`](/pgd/latest/reference/catalogs-visible#bdrstat_subscription). The new columns are `nprovisional_waits`, `ntuple_waits` and `ncommmit_waits`. The `nprovisional_waits` value reflects the number of operations on the same tuples being performed by concurrent apply transactions. These are provisional waits which aren't actually waiting yet but could. 
+To support Parallel Apply's deadlock mitigation, PGD 5.2 adds columns to [`bdr.stat_subscription`](/pgd/latest/reference/catalogs-visible#bdrstat_subscription). The new columns are `nprovisional_waits`, `ntuple_waits` and `ncommmit_waits`. These 
+are metrics which indicate how well Parallel Apply is managing what would have previously been deadlocks. They are not 
+indicative of overall system performance
```

Comment:
Superfluous line break?

Reply:
Is markdown. Is rewrapped as it is formatted for the output.